### ISSUE TITLE
{MediaInfo AudioChannels} file renaming token result is blank on certain DTS-X

### ISSUE THREAD

**Issue by bakerboy448** [Radarr/Radarr#5404]
**Describe the bug**
certain DTS movies not reporting any {MediaInfo AudioChannels}
https://www.reddit.com/r/radarr/comments/k0joca/mediainfo_audiochannels_file_renaming_token/
**To Reproduce**
N/A see media info

**Expected behavior**
Report Audio Channels correctly.

**Screenshots**

**Platform Information (please complete the following information):**
 - OS: N/A
 - Docker: N/A
 - Mono or.NET Core Version: N/A
 - Browser and Version N/A
 - Radarr Version 3.0.0.4142
 - Radarr Branch nightly

**Trace Logs**
Not needed; Media info from mediainfo and DB Below

**Comment by bakerboy448** [comment]
From OP's Radarr Database MediaInfo

Deathly Hallows (problematic):

```
      "mediaInfo": {
        "audioAdditionalFeatures": "XLL X",
        "audioBitrate": 3787224,
        "audioChannels": -0.9,
        "audioCodec": "DTS-X",
        "audioLanguages": "English",
        "audioStreamCount": 1,
        "videoBitDepth": 10,
        "videoBitrate": 21239854,
        "videoCodec": "x265",
        "videoFps": 23.976,
        "resolution": "3840x1600",
        "runTime": "2:10:26",
        "scanType": "",
        "subtitles": "English / French / Italian / Chinese / Chinese / Korean / Spanish / Portuguese / Arabic / Danish / Finnish / Norwegian / Swedish / Japanese"
      },
```

Goblet of Fire (working fine):
```
      "mediaInfo": {
        "audioAdditionalFeatures": "XLL X",
        "audioBitrate": 4051121,
        "audioChannels": 7.1,
        "audioCodec": "DTS-X",
        "audioLanguages": "English",
        "audioStreamCount": 1,
        "videoBitDepth": 10,
        "videoBitrate": 13391939,
        "videoCodec": "x265",
        "videoFps": 23.976,
        "resolution": "3840x1600",
        "runTime": "2:37:5",
        "scanType": "",
        "subtitles": "English / French / German / Italian / Spanish / Spanish / Portuguese / Japanese"
      },
```

**Comment by bakerboy448** [comment]
for posterity, OP's `mediainfo --Language=raw --full` output


Deathly Hallows:

        Audio
        Count                            : 280
        StreamCount                      : 1
        StreamKind                       : Audio
        StreamKind/String                : Audio
        StreamKindID                     : 0
        StreamOrder                      : 1
        ID                               : 2
        ID/String                        : 2
        Format                           : DTS
        Format/String                    : DTS XLL X
        Format/Info                      : Digital Theater Systems
        Format/Url                       : https://en.wikipedia.org/wiki/DTS_(sound_system)
        Format_Commercial                : DTS-HD Master Audio
        Format_Commercial_IfAny          : DTS-HD Master Audio
        Format_Settings_Mode             : 16
        Format_Settings_Endianness       : Big
        Format_AdditionalFeatures        : XLL X
        CodecID                          : A_DTS
        Duration                         : 8765259.000000
        Duration/String                  : 2h 26mn
        Duration/String1                 : 2h 26mn 5s 259ms
        Duration/String2                 : 2h 26mn
        Duration/String3                 : 02:26:05.259
        Duration/String4                 : 02:25:41:89
        Duration/String5                 : 02:26:05.259 (02:25:41:89)
        BitRate_Mode                     : VBR
        BitRate_Mode/String              : VBR
        BitRate                          : 3683214
        BitRate/String                   : 3683 Kbps
        Channel(s)                       : 8
        Channel(s)/String                : 8 channel
        Channel(s)_Original              : Object Based
        Channel(s)_Original/String       : Object Based
        ChannelPositions_Original        : Object Based
        ChannelLayout_Original           : Object Based
        SamplesPerFrame                  : 512
        SamplingRate                     : 48000
        SamplingRate/String              : 48.0 KHz
        SamplingCount                    : 420732432
        FrameRate                        : 93.750
        FrameRate/String                 : 93.750 fps (512 SPF)
        FrameCount                       : 821743
        BitDepth                         : 24
        BitDepth/String                  : 24 bit
        Delay                            : 0
        Delay/String3                    : 00:00:00.000
        Delay_Source                     : Container
        Delay_Source/String              : Container
        Video_Delay                      : 0
        Video_Delay/String3              : 00:00:00.000
        StreamSize                       : 4035541028
        StreamSize/String                : 3.76 GiB (20%)
        StreamSize/String1               : 4 GiB
        StreamSize/String2               : 3.8 GiB
        StreamSize/String3               : 3.76 GiB
        StreamSize/String4               : 3.758 GiB
        StreamSize/String5               : 3.76 GiB (20%)
        StreamSize_Proportion            : 0.19536
        Language                         : en
        Language/String                  : en
        Language/String1                 : en
        Language/String2                 : en
        Language/String3                 : eng
        Language/String4                 : en
        Default                          : Yes
        Default/String                   : Yes
        Forced                           : No
        Forced/String                    : No

Goblet of Fire:

        Audio
        Count                            : 280
        StreamCount                      : 1
        StreamKind                       : Audio
        StreamKind/String                : Audio
        StreamKindID                     : 0
        StreamOrder                      : 1
        ID                               : 2
        ID/String                        : 2
        Format                           : DTS
        Format/String                    : DTS XLL X
        Format/Info                      : Digital Theater Systems
        Format/Url                       : https://en.wikipedia.org/wiki/DTS_(sound_system)
        Format_Commercial                : DTS-HD Master Audio
        Format_Commercial_IfAny          : DTS-HD Master Audio
        Format_Settings_Mode             : 16
        Format_Settings_Endianness       : Big
        Format_AdditionalFeatures        : XLL X
        CodecID                          : A_DTS
        Duration                         : 9425408.000000
        Duration/String                  : 2h 37mn
        Duration/String1                 : 2h 37mn 5s 408ms
        Duration/String2                 : 2h 37mn
        Duration/String3                 : 02:37:05.408
        Duration/String4                 : 02:36:40:32
        Duration/String5                 : 02:37:05.408 (02:36:40:32)
        BitRate_Mode                     : VBR
        BitRate_Mode/String              : VBR
        BitRate                          : 4051121
        BitRate/String                   : 4051 Kbps
        Channel(s)                       : 8
        Channel(s)/String                : 8 channel
        Channel(s)_Original              : Object Based
        Channel(s)_Original/String       : Object Based
        ChannelPositions_Original        : Object Based
        ChannelLayout_Original           : Object Based
        SamplesPerFrame                  : 512
        SamplingRate                     : 48000
        SamplingRate/String              : 48.0 KHz
        SamplingCount                    : 452419584
        FrameRate                        : 93.750
        FrameRate/String                 : 93.750 fps (512 SPF)
        FrameCount                       : 883632
        BitDepth                         : 24
        BitDepth/String                  : 24 bit
        Delay                            : 0
        Delay/String3                    : 00:00:00.000
        Delay_Source                     : Container
        Delay_Source/String              : Container
        Video_Delay                      : 0
        Video_Delay/String3              : 00:00:00.000
        StreamSize                       : 4772934524
        StreamSize/String                : 4.45 GiB (23%)
        StreamSize/String1               : 4 GiB
        StreamSize/String2               : 4.4 GiB
        StreamSize/String3               : 4.45 GiB
        StreamSize/String4               : 4.445 GiB
        StreamSize/String5               : 4.45 GiB (23%)
        StreamSize_Proportion            : 0.23104
        Language                         : en
        Language/String                  : en
        Language/String1                 : en
        Language/String2                 : en
        Language/String3                 : eng
        Language/String4                 : en
        Default                          : Yes
        Default/String                   : Yes
        Forced                           : No
        Forced/String                    : No

**Comment by Qstick** [comment]
@bakerboy448 see if

**Comment by bakerboy448** [comment]
for posterity; closed by #5412
